braindecode/models/sleep_stager_blanco_2020.py

```diff
@@ -1,11 +1,16 @@
 # Authors: Divyesh Narayanan <divyesh.narayanan@gmail.com>
+#          Bruno Aristimunha <b.aristimunha@gmail.com>
 #
 # License: BSD (3-clause)
 
 import numpy as np
 import torch
 from torch import nn
 
+from .util import check_deprecation_warning
+
+pair_new_names_blanco = {"n_conv_chans": "n_conv_channels"}
```

Comment:
I woould move this to inside the constructor, even to inside the if... no need to have this so global I think